src/test/integration/session.spec.ts

```diff
@@ -235,8 +236,7 @@ describe("Session Token", function() {
       consistencyLevel: ConsistencyLevel.Session
     });
 
```

Comment:
This is the actually important part of the PR. It should make the session test not be flaky anymore

Reply:
It looks like the only change here is using the test helper to get the database. If that's it, then yeah; makes sense.